Parsers/ASimAuthentication/ProductParsers/AuthenticationM365D.yaml

```diff
@@ -0,0 +1,131 @@
+Parser:
+  Title: ASIM M365 Defender DeviceLogonEvent Authentication parser
+  Version: '0.1.0'
+  LastUpdated: July 1, 2021
+Product:
+  Name: M365 Defender
+Normalization:
+  Schema: Authentication
+  Version: '0.1.0'
+References:
+- Title: ASIM Authentication Schema
+  Link: https://aka.ms/AzSentinelAuthenticationDoc
+- Title: ASIM
+  Link: https:/aka.ms/AzSentinelNormalization
+Description: |
+      This Query Parser maps M365 Defender Device File Events (DeviceFileEvents) to the Azure Sentinel Information Model authenticaion schema.
+ParserName: vimAuthenticationM365Defender
+ParserQuery: |
+  let FaliureReason=datatable(EventOriginalResultDetails:string, EventResultDetails:string)[
+    'InvalidUserNameOrPassword','No such user or password' 
+    // What other will we find here?
+    ];
+  let AuthM365D=(){
+    DeviceLogonEvents 
+    // ---   Already mapped
+    // // TimeGenerated - exists
+    //
+    | project-rename 
+       EventOriginalResultDetails=FailureReason 
+       , EventSubType=LogonType
```

Comment:
Normalize to the same values you did for SecurityEvent. In practice, it means Remote interactive (RDP) logons => RemoteInteractive

Reply:
since it is only one value that needs change - using if instead of lookup